### ISSUE TITLE
Bug Report: Unsupported installation - cgroup configuration

### ISSUE THREAD

**Issue by johny-mnemonic** [home-assistant/supervised-installer#239]
### OS Version

Debian GNU/Linux 11 (bullseye)

### System Information

Linux hassio 5.10.0-18-arm64 #1 SMP Debian 5.10.140-1 (2022-09-02) aarch64 GNU/Linux

### What happened?

Installer spits the warning below, that it is unable to perform the switch to cgroup v1 and inherently the HA then reports the system unsupported thanks to wrong configuration of cgroups.

The issue is, clean [Debian 11 for RPi](https://raspi.debian.net//) doesn't use grub to boot and it also doesn't have `cmdline.txt` in `/boot` (which is a path used by RaspiOS or Raspbian). Correct path to `cmdline.txt` in clean Debian 11 for RPi is `/boot/firmware/cmdline.txt`

### Machine Type

raspberrypi3-64

### Installer output

```text
[warn] Could not find /etc/default/grub or /boot/cmdline.txt failed to switch to cgroup v1
```

### ADR

- [X] I have read through the [ADR](https://github.com/home-assistant/architecture/blob/master/adr/0014-home-assistant-supervised.md) and have confirmed that my system is compliant with the requirements
- [X] I understand that if my system is found to not be compliant, my issue will be closed immediately without further investigation

### Code of Conduct

- [X] I have read and agree to the [Home Assistant Code of Conduct](https://www.home-assistant.io/code_of_conduct/)

**Comment by johny-mnemonic** [comment]
Seems like this issue was already reported previously in https://github.com/home-assistant/supervised-installer/issues/221 but closed without any attention 😢

**Comment by ikifar2012** [comment]
Should be solved in version 1.3.1 https://github.com/home-assistant/supervised-installer/releases/tag/1.3.1
